model/portableElement/storage/PortableElementRegistry.php

```diff
@@ -363,11 +369,20 @@ protected function getRuntime(PortableElementObject $object)
     }
 
     /**
-     * @return array
-     * @throws PortableElementInconsistencyModelException
+     * Get the alias version for a given version number, e.g. 2.1.5 becomes 2.1.*
+     * @param $versionString
+     * @return mixed
      */
-    public function getLatestRuntimes()
-    {
+    private function getAliasVersion($versionString){
+        return preg_replace('/^([0-9]+\.[0-9]+\.)([0-9]+)$/', '$1*', $versionString);
```

Comment:
It's not a show-stopper, but handling version with a library would be safer (or do we force registration of `x.x.x` version format ?)
If we don't have one already in tao, there's https://github.com/naneau/semver